js/components/LetterImage.js

```diff
@@ -60,13 +79,38 @@ class LetterImage extends React.Component {
       dims = this.resizeKeepAspect(imgWidth, imgHeight, maxDim);
     }
 
+    let margin = MEDIUM_MARGIN;
+
+    let leftExpanded = Math.max(0, this.props.coords.left - margin);
+    let topExpanded = Math.max(0, this.props.coords.top - margin);
+    let rightExpanded = Math.min(this.props.coords.pagewidth, 
+                                 this.props.coords.left + this.props.coords.width + margin);
+    let widthExpanded = rightExpanded - leftExpanded;
+    let bottomExpanded = Math.min(this.props.coords.pageheight,
+                                  this.props.coords.top + this.props.coords.height + margin);
+    let heightExpanded = bottomExpanded - topExpanded;
+
+    let cropURL =
+    "http://127.0.0.1:8000/api/crop?page_url=" +
```

Comment:
Can you switch this to the reclaim url?

Reply:
Yes that's a show-stopper, and for that to happen I need to deploy the API updates to the reclaim backend first... stay tuned.

Reply:
OK I deployed the backend API updates and also updated the cropURL above to use the Reclaim backend, so this should be good to go -- unless the actual LetterImage code for displaying the cropped images seems better left as a WIP.